ChangeLog.d/fix-iar-compiler-warnings.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix IAR compiler warnings. Fixes #7873, #4300.
```

Comment:
Since #7873 is raised against 2.28, how come there isn't a backport (if only for the ChangeLog)?

Reply:
Ahh yes this is my bad, the issues (both of them) are fixes to 2.28, I'll make a backport PR for the Changelog (for reference they were fixed in #8015)